### DISCUSSION TITLE
Cannot find STM32F413VGT6

### DISCUSSION THREAD

**Discussion by basit-eSpark** [stm32duino/Arduino_Core_STM32#2075] [Variants] [answered]
I cannot find STM32F413VGT6 variant in the board part# list. The Generic F413 does not list VGTx variant.

**Comment by fpistm** [comment, marked as answer]
Hi @basit-eSpark 
not all the mcu have been declared.
All generic files are present https://github.com/stm32duino/Arduino_Core_STM32/tree/main/variants/STM32F4xx/F413V(G-H)T_F423VHT
but it still required to declare it and add linker script as described here:
https://github.com/stm32duino/Arduino_Core_STM32/wiki/Add-a-new-variant-%28board%29

Feel free to add it and submit a PR. Any contributions are welcome 😉
